lnrpc/routerrpc/config_active.go

```diff
@@ -40,4 +40,8 @@ type Config struct {
 	//
 	// TODO(roasbeef): assumes router handles saving payment state
 	Router *routing.ChannelRouter
+
+	// RouterBackend contains shared logic between this sub server and the
+	// main rpc server.
+	RouterBackend *RouterBackend
```

Comment:
We only need this for the self node pubkey right? I think we can pass in just that instead.

Reply:
In this PR it is just that one, but my follow up https://github.com/lightningnetwork/lnd/pull/1662 requires access to other parts of it.